ocaml/idl/datamodel_types.ml

```diff
@@ -275,7 +276,14 @@ let release_order_full = [{
      version_minor = 15;
      branding      = "Citrix Hypervisor 8.2 Hotfix 2";
      release_date  = None;
+   }; {
+     code_name     = Some rel_yangtze_https;
+     version_major = 2;
+     version_minor = 16;
```

Comment:
Did we forget to add the yangtze release here? Maybe it wasn't needed because the API didn't change

Reply:
It is not defined; I can add it if we should have it

Reply:
If it's not used in the IDL, it's not needed. I think the IDL lifecycles in master should match these for consistency

Reply:
```